src/ui/public/chrome/api/angular.js

```diff
@@ -71,28 +72,28 @@ export function initAngularApi(chrome, internals) {
           return $location.path().split('/')[1];
         };
 
-        const notify = new Notifier();
         const urlOverflow = Private(UrlOverflowServiceProvider);
         const check = () => {
-        // disable long url checks when storing state in session storage
-          if (config.get('state:storeInSessionStorage')) return;
-          if ($location.path() === '/error/url-overflow') return;
+          // disable long url checks when storing state in session storage
+          if (config.get('state:storeInSessionStorage')) {
+            return;
+          }
+
+          if ($location.path() === '/error/url-overflow') {
+            return;
+          }
 
           try {
             if (urlOverflow.check($location.absUrl()) <= URL_LIMIT_WARN_WITHIN) {
-              notify.directive({
-                template: `
-                <p>
-                  The URL has gotten big and may cause Kibana
-                  to stop working. Please either enable the
-                  <code>state:storeInSessionStorage</code>
-                  option in the <a href="#/management/kibana/settings">advanced
-                  settings</a> or simplify the onscreen visuals.
-                </p>
-              `
-              }, {
-                type: 'error',
-                actions: [{ text: 'close' }]
+              toastNotifications.addWarning({
+                title: 'The URL has gotten big and may cause Kibana to stop working',
+                text: (
+                  <Fragment>
+                    Please either enable the <code>state:storeInSessionStorage</code> option
+                    in the <a href="#/management/kibana/settings">advanced settings</a> or
+                    simplify the onscreen visuals
+                  </Fragment>
```

Comment:
The URL is big and Kibana might stop working

Either enable the  <code>state:storeInSessionStorage</code> option in <a href="#/management/kibana/settings">advanced settings</a> or simplify the onscreen visuals.